content/pom.xml

```diff
@@ -289,7 +289,7 @@
                                         <replace file="target/jcr_root/libs/core/wcm/components/separator/v1/separator/.content.xml" token=".core-wcm" value="Core Content" />
                                         <replace file="target/jcr_root/libs/core/wcm/components/sharing/v1/sharing/.content.xml" token=".core-wcm" value="Core Content" />
                                         <replace file="target/jcr_root/libs/core/wcm/components/tabs/v1/tabs/.content.xml" token=".core-wcm" value="Core Content" />
-                                        <replace file="target/jcr_root/libs/core/wcm/components/teaser/v1/teaser/.content.xml" token=".core-wcm" value="Core Content" />
+                                        <replace file="target/jcr_root/libs/core/wcm/components/teaser/v2/teaser/.content.xml" token=".core-wcm" value="Core Content" />
```

Comment:
We should not expose this in a public group until it's ready for production

Reply:
Addressed in https://github.com/adobe/aem-core-wcm-components/pull/1426/commits/7915c57a2aaa371ba2464d8d46b6587c695effc9